contracts/Marketplace.sol

```diff
@@ -1,5 +1,5 @@
 // solhint-disable not-rely-on-time
-pragma solidity ^0.5.16;
+pragma solidity ^0.6.6;
```

Comment:
Solidity has been in 8 since late last year; might as well jump directly there. Unless of course it would require lots of rework for some reason.

Reply:
uniswap is solc6